app/components/layout/Navigation/support-dialog.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import ReactCSSTransitionGroup from 'react-addons-css-transition-group';
+import shuffle from 'shuffle-array';
+
+import Button from '../../shared/Button';
+import Emojify from '../../shared/Emojify';
+import Icon from '../../shared/Icon';
+
+const PEOPLE = [
+  { image: "keithpitt", name: "Keith Pitt" },
+  { image: "harriet", name: "Harriet Lawrence" },
+  { image: "ticky", name: "Jessica Stokes" },
+  { image: "sj26", name: "Sam Cochran" },
+  { image: "toolmantim", name: "Tim Lucas" }
+];
+
+class SupportDialog extends React.Component {
+  static displayName = "Navigation.SupportDialog";
+
+  static propTypes = {
+    onClose: React.PropTypes.func
+  };
+
+  state = {
+    showing: false
+  };
+
+  // Using setState in here is a bit yucky, but it's the easiest way to get
+  // ReactCSSTransitionGroup to behave correctly so it shows the animation when
+  // mounting.
+  componentDidMount() {
+    /*eslint-disable react/no-did-mount-set-state */
+    setTimeout(() => {
+      this.setState({ showing: true });
+    }, 0);
+    /*eslint-enable react/no-did-mount-set-state */
+  }
```

Comment:
Since we’re already managing state in the parent component, maybe the transition group should in fact go there? Seems to be the approach used by `BuildsDropdown`